src/templates/404-page.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { Grid, GridContainer, Button } from '@trussworks/react-uswds';
+import { graphql, navigate } from 'gatsby';
+import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+
+import { Layout } from '../components';
+import { page404 as links } from '../constants/links';
+
+function Page404({ data }) {
+  const { i18n } = useTranslation();
+
+  const { markdownRemark } = data;
+  const { frontmatter } = markdownRemark;
+
+  return (
+    <>
+      <Helmet title={frontmatter.pageTitle} htmlAttributes={{ lang: i18n.language }} />
+      <Layout slug={frontmatter.slug}>
+        <main>
+          <section className="usa-section">
```

Comment:
You can use the section component that already exists, as we did in the landing page. Or perhaps just use the markdown body for the text sections, like in the documentation page

Reply:
I see what you mean, but the contents inside the section component don't match the content for the 404 pages. For example, we would need multiple text and buttons, while the section component only has placeholders for one text and one button. We could use multiple sections for the 404 page to accomplish this, but it wouldn't make sense to use multiple sections when the 404 content is all related. 

Also, there is an `h1` tag on the 404 pages, but the section component does not have one. We can totally use markdown body for the text sections, but by doing it the current way, we're making the 404 pages more easily editable.

Reply:
I think using markdown body would be more easily editable, since you can clearly see the text layout

Reply:
Markdown body would make sense for something like the documentation where it's all text. However, in cases like this where code is also incorporated, it would be best to go about it the way we do now. Otherwise, we would have used the markdown body for the landing pages to start with.